package.json

```diff
@@ -27,5 +37,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "enzyme": "^3.10.0",
```

Comment:
Once you decided to adopt react `hook`, there will be a question for how to test? If `enzyme` is not supporting well, maybe you could give [react-testing-library](https://github.com/testing- library/react-testing-library) a try 🙌

Reply:
Thanks for the advice, I'll go check on that:)